pygeoapi/provider/__init__.py

```diff
@@ -35,7 +35,9 @@
 PROVIDERS = {
     'CSV': 'pygeoapi.provider.csv_.CSVProvider',
     'Elasticsearch': 'pygeoapi.provider.elasticsearch_.ElasticsearchProvider',
-    'GeoJSON': 'pygeoapi.provider.geojson.GeoJSONProvider'
+    'GeoJSON': 'pygeoapi.provider.geojson.GeoJSONProvider',
+    'Sqlite': 'pygeoapi.provider.sqlite.SQLiteProvider',
```

Comment:
Should we call this `SQLite` for consistency?

Reply:
Will change it